__tests__/Unit/Components/ProgressForm/ProgressHeader.test.tsx

```diff
@@ -0,0 +1,21 @@
+import { render, screen } from '@testing-library/react';
+
+import ProgressHeader from '@/components/ProgressForm/ProgressHeader';
+
+describe('ProgressHeader', () => {
+    test('Should render Text in Progress Header component', () => {
+        render(<ProgressHeader totalMissedUpdates={10} updateType="Standup" />);
+        const missedUpdatesElement = screen.getByText(
+            // eslint-disable-next-line quotes
```

Comment:
Please avoid using this.

Reply:
https://github.com/Real-Dev-Squad/website-status/pull/710#discussion_r1264517747

Reply:
No, let's figure out a way to avoid this.